app/reducers/trezor.js

```diff
@@ -99,7 +99,8 @@ export default function trezor(state = {}, action) {
       return {
         ...state,
         device: null,
-        deviceLabel: null
+        deviceLabel: null,
+        connected: false
```

Comment:
This was needed. Otherwise pulling the device will put us in a state where we are connected but have no device.

Reply:
might be good dropping a comment into the code?

Reply:
might be good dropping a comment into the code?

Reply:
Ok.